spring-soap/pom.xml

```diff
@@ -38,26 +38,6 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
-            <!-- tag::xsd[] -->
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>jaxb2-maven-plugin</artifactId>
-                <version>1.6</version>
-                <executions>
-                    <execution>
-                        <id>xjc</id>
-                        <goals>
-                            <goal>xjc</goal>
-                        </goals>
-                    </execution>
-                </executions>
-                <configuration>
-                    <schemaDirectory>${project.basedir}/src/main/resources/</schemaDirectory>
-                    <outputDirectory>${project.basedir}/src/main/java</outputDirectory>
-                    <clearOutputDir>false</clearOutputDir>
-                </configuration>
-            </plugin>
-            <!-- end::xsd[] -->
```

Comment:
Should this xsd plugin be removed? It might impact another tutorial in this module.

Reply:
indeed, reverted the changes: https://github.com/eugenp/tutorials/pull/12581/commits/54136e44032c9377b82a353aebf4756773d2ad4f